addons/website_slides_survey/tests/test_course_ui_certification.py

```diff
@@ -0,0 +1,69 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+from odoo.addons.base.tests.common import HttpCaseWithUserDemo
+
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestCourseUiCertification(HttpCaseWithUserDemo):
+
+    def setUp(self):
+        super().setUp()
+
+        self.survey_certification = self.env['survey.survey'].create({
```

Comment:
Maybe stupid question, but we have some tests with certifications in "test website slides full", can't we include that step in those and avoid adding a new tour ? Or do you prefer having a specific tour ?

Reply:
Not a stupid question at all, did not think about this module.

Had to tweak steps a bit but in the end it helps saving a whole lot of data preparation and lowers the diff, thanks!